airbyte-integrations/connectors/source-retently/source_retently/source.py

```diff
@@ -201,32 +211,3 @@ def parse_response(
     ) -> Iterable[Mapping]:
         data = response.json().get("data")
         yield data
-
-class Campaigns(RetentlyStream):
-    json_path = "campaigns"
-
-    def path(
-        self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
-    ) -> str:
-        return "campaigns"
-
-    def parse_response(
-        self,
-        response: requests.Response,
-        stream_state: Mapping[str, Any],
-        stream_slice: Mapping[str, Any] = None,
-        next_page_token: Mapping[str, Any] = None,
-    ) -> Iterable[Mapping]:
-        data = response.json()
-        stream_data = data.get(self.json_path) if self.json_path else data
-        for d in stream_data:
-            yield d
-
-class Feedback(RetentlyStream):
```

Comment:
Why you removed this 2 streams?

Reply:
Campaigns and Feedback streams are already implemented in the file, these two classes were a redundant result of a bad merge.